src/azure-cli/azure/cli/command_modules/storage/_params.py

```diff
@@ -207,8 +207,9 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
 
     allow_cross_tenant_replication_type = CLIArgumentType(
         arg_type=get_three_state_flag(), options_list=['--allow-cross-tenant-replication', '-r'], min_api='2021-04-01',
-        help='Allow or disallow cross AAD tenant object replication. The default interpretation is true for this '
-        'property.')
+        help='Allow or disallow cross AAD tenant object replication. Set this property to true for new or existing '
+             'accounts only if object replication policies will involve storage accounts in different AAD tenants. '
+             'The default interpretation is false for new accounts to follow best security practices by default.')
```

Comment:
`The default interpretation is ... by default` sounds awkward

Reply:
Updated